### PR TITLE
Add pd-ctl  `balance-leader-scheduler` config command

### PR DESCRIPTION
**Pull request by en-jin19**
### What is changed, added or deleted? (Required)
Add pd-ctl  `balance-leader-scheduler` configuration command

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/8677
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by en-jin19** [comment]
Hi @[CabinfeverB](https://github.com/CabinfeverB), could you please tech review this PR? thank you!

File: pd-control.md [hunk 1]
```diff
@@ -742,6 +742,7 @@ Usage:
 >> scheduler resume balance-region-scheduler      // Continue to run the balance-region scheduler
 >> scheduler resume all                           // Continue to run all schedulers
 >> scheduler config balance-hot-region-scheduler  // Display the configuration of the balance-hot-region scheduler
+>> scheduler config balance-leader-scheduler set batch 3 // Set the size of the operators that the balance-leader scheduler can execute in batches to 3
```
Comment (en-jin19) [inline pd-control.md:745]:
```suggestion
>> scheduler config balance-leader-scheduler set batch 3 // Set 3 as the size of the operators that the balance-leader scheduler can execute in batches
```

**Review by CabinfeverB — APPROVED** [review]
LGTM

File: pd-control.md [hunk 2]
```diff
@@ -742,6 +742,7 @@ Usage:
 >> scheduler resume balance-region-scheduler      // Continue to run the balance-region scheduler
 >> scheduler resume all                           // Continue to run all schedulers
 >> scheduler config balance-hot-region-scheduler  // Display the configuration of the balance-hot-region scheduler
+>> scheduler config balance-leader-scheduler set batch 3 // Set 3 as the size of the operator that the balance-leader scheduler can execute in batches
```
Comment (TomShawn) [inline pd-control.md:745]:
```suggestion
>> scheduler config balance-leader-scheduler set batch 3 // Set the size of the operator that the balance-leader scheduler can execute in a batch to 3
```

**Review by TomShawn — APPROVED** [review]
Rest LGTM

**Pull request merged**